test/functional/rpc_getblockstats.py

```diff
@@ -46,17 +46,19 @@ def generate_test_data(self, filename):
 
         address = self.nodes[0].getnewaddress('', 'bech32')
         self.nodes[0].sendtoaddress(address=address, amount=10, subtractfeefromamount=True)
+        self.nodes[0].sendtoaddress(address=address, amount=10, subtractfeefromamount=True)
         self.nodes[0].generate(1)
         self.sync_all()
 
-        self.nodes[0].sendtoaddress(address=address, amount=10, subtractfeefromamount=True)
-
         inputs = self.nodes[0].listunspent()[:3]
-        outputs = {self.nodes[0].getnewaddress():1,self.nodes[0].getnewaddress():2,self.nodes[0].getnewaddress():sum(i["amount"] for i in inputs)-Decimal("3.001")}
+        outputs = {self.nodes[0].getnewaddress():sum(i["amount"] for i in inputs)-Decimal("0.001")}
```

Comment:
I think these changes to the test file can go in the previous commit (_Add more stats to getblockstats_) so the consolidation transaction stat is triggered in that commit.